src/contants/actionTypes.js

```diff
@@ -0,0 +1,3 @@
+export const ADD_DATA = 'ADD_DATA'; // dummy constant
```

Comment:
This looks good to me. But the file name 'contants' is not descriptive. I believe it would be better to have actionTypes under the src directory.

Reply:
Thanks for pointing that. That is suppose to be `constants`, I'll make an update on that. Regarding the location, the idea is for us to have every constant variables in a single directory.

Reply:
Good!